access/mattermost/mattermost_test.go

```diff
@@ -27,6 +28,8 @@ import (
 )
 
 var msgFieldRegexp = regexp.MustCompile(`(?im)^\*\*([a-zA-Z ]+)\*\*:\ +(.+)$`)
+var requestReasonRegexp = regexp.MustCompile(`(?im)^\*\*Reason\*\*:\ ` + "```\\n" + `(.*?)` + "```" + `(.*?)$`)
```

Comment:
nit: did you separate this into `` and "" parts to avoid using \\\\s? I'd prefer a single string with \\\\s personally.
```suggestion
var requestReasonRegexp = regexp.MustCompile("(?im)^\\*\\*Reason\\*\\*:\\ ```\\n(.*?)```(.*?)$")
```